src/main/frontend/src/components/HarvesterAdmin.vue

```diff
@@ -189,18 +189,48 @@ async function removeInstitution(anInstitutionID) {
     <v-dialog v-model="displayInstitutionForm" width="768">
         <v-card>
             <v-form>
-                <v-text-field v-if="institutionToAddOrUpdate.id" label="ID" v-model="institutionToAddOrUpdate.id" disabled></v-text-field>
+                <v-text-field
+                    v-if="institutionToAddOrUpdate.id"
+                    label="ID"
+                    v-model="institutionToAddOrUpdate.id"
+                    disabled
+                ></v-text-field>
```

Comment:
I was wondering if that hanging `>` was a standard formatting. Good to know, thanks.

Reply:
It's a combo of the line wrapping due to being longer than the configured `printWidth`, and [this rule](https://prettier.io/docs/en/options.html#bracket-line) (which I'd just left as the default).